tests/integration/helpers.py

```diff
@@ -70,6 +70,19 @@ def oci_image(metadata_file: str, image_name: str) -> str:
     return upstream_source
 
 
+async def loki_rules(ops_test, app_name) -> dict:
+    address = await get_unit_address(ops_test, app_name, 0)
+    url = f"http://{address}:3100"
+
+    try:
+        response = urllib.request.urlopen(f"{url}/loki/api/v1/rules", data=None, timeout=2.0)
+        if response.code == 200:
+            return yaml.safe_load(response.read())
```

Comment:
I think this should test that the rules have the content we expect them to have

Reply:
that feels like a unit test, and something to be addressed in another PR.
The scope of this PR is to add integration tests to verify that 
1) the loki push api does what it should
2) promtail can forward file logs
3) promtail can forward syslog logs